src/test/fuzz/versionbits.cpp

```diff
@@ -281,47 +260,46 @@ FUZZ_TARGET_INIT(versionbits, initialize)
 
     // since is straightforward:
     assert(since % period == 0);
-    assert(0 <= since && since <= current_block->nHeight + 1);
+    assert(0 <= since && since <= height);
     if (state == exp_state) {
         assert(since == exp_since);
     } else {
-        assert(since == current_block->nHeight + 1);
+        assert(since == height);
     }
 
     // state is where everything interesting is
     switch (state) {
     case ThresholdState::DEFINED:
         assert(since == 0);
         assert(exp_state == ThresholdState::DEFINED);
-        assert(current_block->GetMedianTimePast() < checker.m_begin);
-        assert(current_block->GetMedianTimePast() < checker.m_end);
+        assert(never_active_test || height < checker.m_begin);
         break;
     case ThresholdState::STARTED:
-        assert(current_block->GetMedianTimePast() >= checker.m_begin);
-        assert(current_block->GetMedianTimePast() < checker.m_end);
+        assert(height >= checker.m_begin);
```

Comment:
In commit "Migrate versionbits to use height instead of MTP":

You could incorporate the later `if (never_active_test) assert(state == ThresholdState::DEFINED);` here as `assert(!never_active_test);`.

(I found this by drawing the finite state machine based on these tests, and noticed this line was missing before noticing it's done explicitly later)

Reply:
Added the new assert, but I did not remove the later assert.